SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -23,14 +23,32 @@ struct ConfigureConnection
 		: Config(InConfig)
 		, Params()
 		, WorkerType(*Config.WorkerType)
-		, ProtocolLogPrefix(*FormatProtocolPrefix())
+		, WorkerLogPrefix(*FormatLogFilePrefix())
 	{
 		Params = Worker_DefaultConnectionParameters();
 
 		Params.worker_type = WorkerType.Get();
 
-		Params.enable_protocol_logging_at_startup = Config.EnableProtocolLoggingAtStartup;
-		Params.protocol_logging.log_prefix = ProtocolLogPrefix.Get();
+		Logsink.logsink_type = WORKER_LOGSINK_TYPE_ROTATING_FILE;
+		Logsink.rotating_logfile_parameters.log_prefix = WorkerLogPrefix.Get();
+		Logsink.rotating_logfile_parameters.max_log_files = 1;
+		Logsink.rotating_logfile_parameters.max_log_file_size_bytes = 10485760; // 10 MB
+
+		uint32_t Categories = 0;
+		if (Config.EnableOpLogging)
+		{
+			Categories |= WORKER_LOG_CATEGORY_API;
+		}
+		if (Config.EnableProtocolLogging)
+		{
+			Categories |= WORKER_LOG_CATEGORY_NETWORK_STATUS | WORKER_LOG_CATEGORY_NETWORK_TRAFFIC;
+		}
+		Logsink.filter_parameters.categories = Categories;
+		Logsink.filter_parameters.level = WORKER_LOG_LEVEL_INFO;
```

Comment:
Users should be able to configure this no? We allow configuration for `WorkerLogs` in the current iteration (logs generated by the worker which are forwarded to the runtime). So I believe this should also be a config value.

Reply:
True, will add that